model/task/task.go

```diff
@@ -3448,6 +3452,21 @@ func (t *Task) SetDisplayTaskID(id string) error {
 		}}))
 }
 
+// SetCheckRunId sets the checkRunId for the task
+func (t *Task) SetCheckRunId(checkRunId int) error {
```

Comment:
Is this supposed to be unit tested? Or is this also going to be tested as part of DEVPROD-970?

Or is the `checkRunHandler` already tested, maybe? Can we add to that test?

Reply:
added a unit test